.github/actions/test-khiops-install/action.yml

```diff
@@ -1,23 +1,93 @@
   steps:
-    - name: Print status for debugging
-      shell: bash
-      # We force the process number to 4 because the runner has only 2 proc.
-      # With 2 proc khiops-env switches to serial and mpi is not used.
+    - name: Test that the executables are installed (Linux/macOS)
+      # We call khiops_env a first time to test the calling of _khiopsgetprocnumber
+      # then we play with KHIOPS_PROC_NUMBER:
+      # - with KHIOPS_PROC_NUMBER=2, khiops should not use MPI, we check that KHIOPS_MPI_COMMAND is empty
+      # - with KHIOPS_PROC_NUMBER=8, khiops should use MPI, we check that KHIOPS_MPI_COMMAND is not empty and KHIOPS_MPI_ERROR is empty
+      #
       # The env var OMPI_MCA_rmaps_base_oversubscribe is for openmpi it corresponds to
       # the flag --oversubscribe (Nodes are allowed to be oversubscribed)
+      if: runner.os != 'Windows'
+      shell: bash
       env:
-        KHIOPS_PROC_NUMBER: 4
         OMPI_MCA_rmaps_base_oversubscribe: true
       run: |
-        khiops-env --env
-    - name: Check Khiops output
+        echo
+        echo --- Khiops environment with KHIOPS_PROC_NUMBER not set
+        khiops_env --env
+        echo
+        export KHIOPS_PROC_NUMBER=2
+        echo --- Khiops environment with KHIOPS_PROC_NUMBER=$KHIOPS_PROC_NUMBER
+        source khiops_env --env
+        if [ ! -z "$KHIOPS_MPI_COMMAND" ]; 
+        then 
+          echo "::error::MPI is used even though there are only 2 procs available"
+          false
+        fi
+        echo
+        export KHIOPS_PROC_NUMBER=8
+        echo --- Khiops environment with KHIOPS_PROC_NUMBER=$KHIOPS_PROC_NUMBER
+        source khiops_env --env
+        if [ ! -z "$KHIOPS_MPI_ERROR" ]; 
+        then 
+          echo "::error::Unexpected MPI error: $KHIOPS_MPI_ERROR"
+          false
+        fi
+        if [ -z "$KHIOPS_MPI_COMMAND" ]; 
+        then 
+          echo "::error::KHIOPS_MPI_COMMAND is not set"
+          false
+        fi
+        echo
+        echo --- Khiops status
+        khiops -s
+        echo
+        echo --- Khiops Coclustering status
+        khiops_coclustering -s
+    - name: Test that the executables are installed (windows)
+      if: runner.os == 'Windows'
+      shell: cmd /C call {0}
+      run: |
+        echo.
+        echo --- Khiops environment with KHIOPS_PROC_NUMBER not set
+        call khiops_env --env
+        echo.
+        set KHIOPS_PROC_NUMBER=2
+        echo --- Khiops environment with KHIOPS_PROC_NUMBER=%KHIOPS_PROC_NUMBER%
+        call khiops_env --env
+        call khiops_env
+        if not "%KHIOPS_MPI_COMMAND%". == "". (
+          echo "::error::MPI is used even though there are only 2 procs available"
+          exit 1
+        )
+        set KHIOPS_PROC_NUMBER=8
+        echo.
+        echo --- Khiops environment with KHIOPS_PROC_NUMBER=%KHIOPS_PROC_NUMBER%
+        call khiops_env --env
+        call khiops_env
+        if not "%KHIOPS_MPI_ERROR%".=="". (
```

Comment:
Why do we need dots here (is it because of carriage returns inside the left-hand side)?
Add space before `==`.